src/configs/chalk.c42spec

```diff
@@ -453,6 +453,173 @@ failure, and no other keys will be checked.
   }
 }
 
+linguist_types := ["programming", "data", "markup"]
+object linguist_language {
+  user_def_ok:   false
+  gen_fieldname: "linguist_languages"
+  gen_typename:  "LinguistLanguage"
+  gen_setters:   false
+
+  field type {
+    type:    string
+    require: true
+    choice: linguist_types
+  }
+
+  field extension {
+    type:    string
+    require: true
+  }
+
+  field altextensions {
+    type:    list[string]
+    require: false
+  }
+
+}
+
+
+valid_tech_stack_categories := ["database", "webServer", "protocol", "framework"]
+valid_tech_stack_database_types := ["firebird", "hypersonicSQL",
```

Comment:
not all of these are used. for example this one dont see any usage. is that on purpose?